### PR TITLE
Allow to filter SigningService by name

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: pulpcore/app/viewsets/content.py [hunk 1]
```diff
@@ -138,3 +149,4 @@ class SigningServiceViewSet(NamedModelViewSet,
     endpoint_name = 'signing-services'
     queryset = SigningService.objects.all()
     serializer_class = SigningServiceSerializer
+    filterset_class = SigningServiceFilter
```
Comment (lubosmj) [inline pulpcore/app/viewsets/content.py]: How about using `filterset_fields` (https://django-filter.readthedocs.io/en/master/guide/rest_framework.html#using-the-filterset-fields-shortcut)? Then you do not need to create a separate class.
```suggestion
    filterset_fields = ['name']
```

File: pulpcore/app/models/content.py [hunk 2]
```diff
@@ -459,7 +459,10 @@ def sign(self, filename):
 
         """
         completed_process = subprocess.run(
-            [self.script, filename], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+            [self.script, filename],
+            env={},
```
Comment (mdellweg) [inline pulpcore/app/models/content.py]: I added this, because the script inherited the complete environment of Django.

**Comment by bmbouter** [comment]
@mdellweg Can we add a changelog entry for this? Since it's a new feature I try to have an entry for any non-trivial change. To do that file a story (even a short one is fine) and then use that number for your commit and CHANGES entry please.

File: pulpcore/plugin/models/__init__.py [hunk 3]
```diff
@@ -24,5 +24,6 @@
     RepositoryContent,
     RepositoryVersion,
     RepositoryVersionDistribution,
+    SigningService,
```
Comment (bmbouter) [inline pulpcore/plugin/models/__init__.py]: This probably needs it's own changelog entry also please.

**Comment by mdellweg** [comment]
I will add tickets and changelogs next week, when I'm back in the office.
If it turns out to be urgent, please take any action needed.

**Comment by bmbouter** [comment]
@mdellweg next week sounds fine. My interest was more to make it clear to anyone coming across the PR that it's status is reviewed and waiting for changelog changes. Thank you!

**Comment by mdellweg** [comment]
ping @bmbouter

**Pull request merged**